xcodeproj/internal/xcschemes/xcscheme_infos.bzl

```diff
@@ -73,7 +73,7 @@ def _make_launch_target(
 
 def _make_same_as_run_launch_target(run_launch_target):
     return struct(
-        extension_host = run_launch_target.extension_host,
+        extension_host = None if run_launch_target.is_path else run_launch_target.extension_host,
```

Comment:
Nit: Sorry if I misunderstood the intent here but IIUC the idea is to set it only if `extension_host` exists correct?  In that case wouldn't `hasattr` be a more direct check here instead of an indirect check via `.is_path`?

Reply:
That's more expensive though. `is_path` is a type check. We did a similar thing elsewhere.

Reply:
Oh ok SGTM, thx for clarifying!

Reply:
updated this